src/status_im/translations/ko.cljs

```diff
@@ -112,33 +108,30 @@
    :browsing-open-in-web-browser          "웹 브라우저로 열기"
```

Comment:
please keep this as @browse as it is a command

Reply:
In line 111 it says @browse

Reply:
ok cool. Are you done with all the reviews and other comments?

Reply:
Yeah, I'll wait for responses from @frostwolfpc. In the meantime I will reflect the suggested changes.